product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Tagging AWS resources"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to cloud provider resources created on your own cloud account (Your Cloud Account). Resource tagging allows you to have fine-grained access control and granular cost data at the bucket level. When you add a tag, the following resources are labeled: 
+
+- EC2 instances
+- S3 buckets
+- Load balancers
+- Volumes
+
+You can create and manage tags on a per-project basis in the EDB Postgres® AI Console. The Console then applies the tag to the resources of all regions configured in your account. 
+
+## Create Tags
+
+To create a tag:
+
+1.  Log in to the Console.
+    
+1.  Go to the project's page.
+
+1.  In the project's list, select your **Project**.
+
+1.  On your project's home page left navigation, select **Cloud Providers**.
+
+1.  On the cloud provider's page, select **Your Cloud Account**.
+
+1.  On your cloud account tab, under **AWS**, select **Manage Tags**.
+
+1.  Select **Add a Key Value pair**.
+
+1.  Provide **Key** and **Value** and select **Save**.
+
+1.  A **Confirm changes to AWS tags** message pops up.
+
+1.  Select **Confirm** to add the tag.
+
+1.  This **Key:Value** pair is added at your project level. It is populated on the EC2 instances, S3 buckets, load balancers and volumes managed in that project, for all regions and clusters.
+
+This new tag appears in the list under **AWS** on Cloud Service Providers. 
+
+This tag is also propagated on your AWS resources. For instance, you can view this tag in the **Tags** tab of your EC2 instance resources on the AWS console.
+
+### Considerations
+
+Consider following while adding a **Key:Value** pair:
+
+- Review the [tagging limitations of AWS](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/tagging-restrictions.html).
+
+- The **Key** cannot have `aws` or `managedby` in the prefix, as they are reserved by AWS and EDB Cloud Service. 
```

Comment:
- `aws:` and `AWS:` both are the reserved prefixes(meaning user can't create tag with key`aws:Team`)
  - `ManagedBy` is the reserved tag key (user can create tag key with `ManagedByABC`)